### DISCUSSION TITLE
Cross-seed with qbitorrent

### DISCUSSION THREAD

**Discussion by m-batista** [cross-seed/cross-seed#128] [Q&A] [answered]
Hello, my torrents are in qbitorrent and i want to re-seed torrents in rtorrent, but i don't know how to do it via qbitorrent, when my torrents in rtorrent i use .rc file or when in deluge i use state file but i don't know which file i should use as torrent dir for qbitorrent

**Comment by mmgoodnow** [comment, marked as answer]
This isn't currently supported, unfortunately. Follow #53 to see when this will be supported
